css/styles.css

```diff
@@ -0,0 +1,848 @@
+  text-transform: uppercase;
+  font-size: 1.5rem;
+  font-weight: 400;
+}
+
+.yellow-bar {
+  background-color: var(--Yellow);
+  transition: all 0.4s;
+}
+
+.yellow-bar a {
+  color: black;
+  font-size: 1.4rem;
+  font-weight: 400;
+}
+
+.feature-article {
+  display: grid;
+  grid-template-areas:
+    "feature-main"
+    "feature-side1"
+    "feature-side2";
+  grid-gap: 0.8rem;
+  overflow: hidden;
+  margin: 0 0.5rem;
+}
+
+.feature-main {
+  grid-area: feature-main;
+  width: 100%;
+  height: 30rem;
+  background:
+    linear-gradient(
+      to left bottom,
+      rgba(151, 18, 18, 0.8) 10%,
+      rgba(0, 0, 0, 0.8) 60%,
+      rgba(0, 0, 0, 1)
+    ),
+    url("https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2019/08/bitcoin_today-header_bitcoin_today-796x419.jpg") no-repeat;
+  background-size: cover;
+  padding: 2rem;
+  color: #fff;
+  position: relative;
+}
+
+.feature-main__top {
+  display: flex;
+  justify-content: space-between;
+}
+
+.feature-article__logo {
+  display: none;
+}
+
+.feature-article__date {
+  font-size: 1.4rem;
+  color: var(--faint-yellow);
+}
+
+.feature-article__bottom {
+  position: absolute;
+  left: 2rem;
+  bottom: 2rem;
+  width: 95%;
+}
+
+.feature-article__h2 {
+  font-size: 3rem;
+  line-height: 1.1;
+  letter-spacing: -0.1rem;
+  margin-bottom: 1.5rem;
+}
+
+.feature-article__h2--side {
+  color: #fff;
+  font-size: 3rem;
+}
+
+.feature-article__tag {
+  text-transform: uppercase;
+  color: #fff;
+  font-size: 1.3rem;
+}
+
+.feature-article__meta,
+.feature-article__meta a {
+  color: var(--grey-light-1);
+  font-size: 1.2rem;
+}
+
+.feature-side1,
+.feature-side2 {
+  width: 100%;
+  height: 30rem;
+}
+
+.feature-side1 {
+  position: relative;
+  background: var(--image-overlay), url(../img/Sprint_image.jpg) no-repeat;
+  background-size: cover;
+  grid-area: feature-side1;
+  padding: 1rem;
+}
+
+.feature-side2 {
+  position: relative;
+  background:
+    var(--image-overlay),
+    url(../img/customer-journey-796x417.png) no-repeat;
+  background-size: cover;
+  grid-area: feature-side2;
+}
+
+.main {
+  max-width: 93%;
+  margin: 0 auto;
+}
+
+.main-heading {
+  width: 100%;
+  margin: 5.5rem 0 1.5rem 0;
+
+  /* display: flex; */
+  justify-content: space-between;
+  align-items: center;
+}
+
+.news-heading a {
+  font-size: 2.7rem;
+  color: var(--Red);
+}
+
+.news {
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-gap: 2rem;
+}
+
+.news-item__img {
+  width: 100%;
+}
+
+.news-item__heading {
+  margin: 0.5rem 0;
+}
+
+.news-item__heading a {
+  color: var(--black);
+  font-size: 1.6rem;
+  line-height: 1.2;
+}
+
+.news__links {
+  display: block;
+}
+
+.news__links li {
+  display: inline-block;
+  margin-left: 1rem;
+}
+
+.news__links a {
+  color: var(--Red);
+  font-size: 1.4rem;
+}
+
+.news-item__list li {
+  color: var(--black);
+  margin-bottom: 1rem;
+}
+
+.news-item {
+  position: relative;
+}
+
+.news-mobile .news-item {
+  display: grid;
+  grid-template-columns: 1fr 3fr;
```

Comment:
on medium screens, in `Latest News` and `Latest Deals` your having some space gap alignment issues between the picture and title description
`grid-template-columns: 0.5fr 3fr;` --> should fix it

Reply:
Thank you @rammazzoti2000 for your keen eye 👍 on details.
The issue was fixed using the your recommendations ⭐.